content/2021/03-16-new-collection-events.md

```diff
@@ -0,0 +1,32 @@
+---
+applied_at: "2021-03-16"
+applies_to: 
+- api
+is_impactful: false
+is_new_feature: true
+collapse: true
+show_excerpt: true
+release_source_url: ''
+---
+
+# New Collection Events
+
+External collection events are now available via the
+[enterprise events][enterprise-events] stream.
+
+[Collections][collections] in Box are a way to group files, folders, and web
+links without putting them all into a folder together. These new events provide
```

Comment:
recommend changing "web links" to "bookmarks" as this is how they are referenced in the web app.